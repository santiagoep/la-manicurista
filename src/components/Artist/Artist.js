import React from "react";

import StyledArtist from "./Artist.styled";

const Artist = ({ name, ...rest }) => (
  <StyledArtist {...rest}>{name}</StyledArtist>
);

export default Artist;
