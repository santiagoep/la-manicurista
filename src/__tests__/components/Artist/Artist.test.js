import React from "react";

import render from "../../render";
import Artist from "@components/Artist/Artist";
import artistsMock from "@__mocks__/components/artists.mock";

describe("<Artist />", () => {
  it("should render snapshot", () => {
    const { renderer } = render({ theme: true, redux: true });
    const component = renderer(<Artist {...artistsMock[0]} />);
    expect(component).toMatchSnapshot();
  });
});
