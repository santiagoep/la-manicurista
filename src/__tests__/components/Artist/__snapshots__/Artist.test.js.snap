// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Artist /> should render snapshot 1`] = `
<p
  className="Artiststyled-w1na2g-0 gTvxEj"
>
  Artists 1
</p>
`;
